spec/factories/access_control_lists.rb

```diff
@@ -2,7 +2,7 @@
 
 FactoryGirl.define do
   factory :access_control_list do
-    user_group
+    association :user_group, factory: :user_group_with_users
```

Comment:
Due to the way the factories are currently set up, this results in `ActiveRecord::RecordInvalid: Validation failed: User group . Roles have already been set for this user or group`. As I said, validations are preventing me from adding factory-built users to an existing ACL. Because the actual specs are in the various shared examples, I can't figure out how to define them in such a way that the specs will pass. Building them in the ACL factory works, though, even if it may cause a bit of unnecessary building elsewhere.